components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIGatewayAdminClient.java

```diff
@@ -479,4 +484,59 @@ public void undeployPolicy(String[] fileNames) throws AxisFault {
             throw new AxisFault("Error occured in removing policy file ", e);
         }
     }
+
+	/**
+     * deploy websocket api to the gateway
+     *
+     * @param content body of the sequence
+     * @param fileName file name of the sequence
+     * @throws AxisFault
+     */
+    public void deployWSApi(String content, String fileName) throws AxisFault {
+        File file = new File(APIConstants.SEQUENCE_FILE_FOLDER);
+        //if directory doesn't exist, make one
+        if (!file.exists()) {
+            file.mkdir();
+        }
+        File writeFile = new File(APIConstants.SEQUENCE_FILE_LOCATION + fileName + APIConstants.XML_EXTENSION);  //file folder+/
+        FileOutputStream fos = null;
+        try {
+            fos = new FileOutputStream(writeFile);
+            //if file doesn't exit make one
+            if (!writeFile.exists()) {
+                writeFile.createNewFile();
+            }
+            byte[] contentInBytes = content.getBytes();
+            fos.write(contentInBytes);
+            fos.flush();
+        } catch (IOException e) {
+
+        } finally {
+            try {
+                if (fos != null) {
+                    fos.close();
+                }
+            } catch (IOException e) {
+                log.error("Error in writing the file" + e, e);
+            }
+        }
+    }
+
+	/**
+	 * remove websocket api from the gateway
+     *
+     * @param fileNames
+     */
+    public void undeployWSApi(String[] fileNames) {
+
+        for (int i = 0; i < fileNames.length; i++) {
+            File file = new File(APIConstants.SEQUENCE_FILE_LOCATION + fileNames[i] + APIConstants.XML_EXTENSION);
```

Comment:
we may take file vairable out side and isn't there any exception thowrs whn delete a file?